docs/essentials/getting-started.md

```diff
@@ -45,4 +47,13 @@ That’s it! You’re now up and running with Normcore. Now, I know this example
 - [XR Avatars and Voice Chat](../guides/xr-avatars-and-voice-chat.md)
 - [Synchronizing Custom Data](../realtime/synchronizing-custom-data.md)
 
-Until next time  : )
\ No newline at end of file
+Until next time  : )
+
+## Normal scoped registry
```

Comment:
This is kind of awkward at the end of the Getting Started guide. It feels a little too advanced for beginners and makes the Getting Started guide feel less simple.

Will you make this into a dedicated page? What do you think is the best place for it? / Where should we put it on the sidebar?

Reply:
I'm thinking:

    Page title: Normal Scoped Registry.
    Section: Essentials. However it's not actually essential and would clutter that section.
    Section: Architecture. It's not related to architecture either but at least it's out of the way.

Or even create a new section? It doesn't 100% fit the existing ones.

Reply:
Architecture works for me for now.